src/state/profile/index.tsx

```diff
@@ -5,7 +5,7 @@ import { getProfile, getProfileAvatar, getUsername } from './helpers'
 
 const initialState: ProfileState = {
   isInitialized: false,
-  isLoading: true,
+  isLoading: false,
```

Comment:
hm... I wonder if it breaks something else somewhere else 🤔

Reply:
It is used in 

1. Profile creation page 
2. User menu
3. IFO
4. Profile page
5. User detail banner in homepage

I see no issues so far